### PR TITLE
VEN-718 | Implement WinterStorageAreaView

### PR DESCRIPTION
**Pull request by ahie**
## Description :sparkles:

Initial implementation for the single winter storage area view.

### Closes :no_good_woman:

[VEN-718](https://helsinkisolutionoffice.atlassian.net/browse/VEN-718)

### Automated tests :gear:️

- `src/common/winterStorageAreaCard/__tests__/WinterStorageAreaCard.test.tsx`
- `src/features/winterStorageAreaView/__tests__/WinterStorageAreaViewContainer.test.tsx`

### Manual testing :construction_worker_man:

1. Open the winter storage areas list page
2. Click on one of the items on the list
3. This should open a new page displaying details for the winter storage area

### CODE REVIEW

File: src/common/winterStorageAreaCard/winterStorageAreaPlaceholder.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (TatuArvela) [inline src/common/winterStorageAreaCard/winterStorageAreaPlaceholder.svg:1]: This same image placeholder is now used in multiple places, and was moved to `/src/common/placeholderImage.svg`

File: src/common/winterStorageAreaCard/WinterStorageAreaCard.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              </Section>
+              <Section>
+                <ExternalLink href={serviceMapUrl} variant="withArrow">
+                  {t('common.terminology.serviceMap')}
+                </ExternalLink>
+              </Section>
+              <HarborMapLinks maps={maps} />
+            </div>
+          </div>
+          <Grid colsCount={6} className={styles.propsGrid}>
```
Comment (TatuArvela) [inline src/common/winterStorageAreaCard/WinterStorageAreaCard.tsx:75]: To make the grid columns of equal width, use this style instead:

```css
grid-template-columns: repeat(6, minmax(0, 1fr));
```

Then add these to Property label styles to make the text wrap and hyphenate:

```css
overflow-wrap: break-word;
width: 100%;
hyphens: auto;
```

Hyphenation will be super weird by default, but you can mark the intended hyphenation places with soft hyphens in the i18n files:

```
"dockingEquipmentSummerStorage": "Telakointi\u00ADtarvikkeiden kesäsäilytys"
```

Comment (ahie) [reply src/common/winterStorageAreaCard/WinterStorageAreaCard.tsx:75]: Added these improvements 👍

File: src/common/winterStorageAreaCard/winterStorageAreaPlaceholder.svg (hunk 1 above)
Comment (ahie) [reply src/common/winterStorageAreaCard/winterStorageAreaPlaceholder.svg]: Switched to using the common placeholder.

File: src/common/winterStorageAreaCard/WinterStorageAreaCard.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+import Card from '../card/Card';
+import styles from '../harborCard/harborCard.module.scss';
```
Comment (TatuArvela) [inline src/common/winterStorageAreaCard/WinterStorageAreaCard.tsx:6]: Shouldn't this be `./winterStorageAreaCard.module.scss`?

Comment (ahie) [reply src/common/winterStorageAreaCard/WinterStorageAreaCard.tsx]: Indeed it should. Fixed.

**Pull request merged**